### DISCUSSION TITLE
v6

### DISCUSSION THREAD

**Discussion by instantgis** [ngx-formly/ngx-formly#2758] [Q&A] [answered]
I'm confused. I see this...

Angular >= 11 | @ngx-formly/core@6.x

I am using Angular 11.0.3 and up until now had been using 5.10.15. So I think sweet let me upgrade.

In the readme and I try to update to @ngx-formly/core@6 but the tag is not there yet so it fails.
I see commit comments about upgrade to Angular 11.

Is it not released yet?
When do we know it is ready for use?

**Comment by aitboudad** [comment, marked as answer]
not ready for production yet, it can be installed using `next` tag https://github.com/ngx-formly/ngx-formly/releases/tag/v6.0.0-next.1
